<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="TestSuite" time="2.61" tests="6" errors="0" skipped="0" failures="0">
    <properties>
        <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
        <property name="sun.boot.library.path" value="/home/kasun/Applications/jdk1.8.0_144/jre/lib/amd64"/>
        <property name="java.vm.version" value="25.144-b01"/>
        <property name="java.vm.vendor" value="Oracle Corporation"/>
        <property name="maven.multiModuleProjectDirectory" value="/home/kasun/git-repos/testgrid"/>
        <property name="java.vendor.url" value="http://java.oracle.com/"/>
        <property name="path.separator" value=":"/>
        <property name="guice.disable.misplaced.annotation.check" value="true"/>
        <property name="java.vm.name" value="Java HotSpot(TM) 64-Bit Server VM"/>
        <property name="file.encoding.pkg" value="sun.io"/>
        <property name="user.country" value="US"/>
        <property name="sun.java.launcher" value="SUN_STANDARD"/>
        <property name="sun.os.patch.level" value="unknown"/>
        <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
        <property name="user.dir" value="/home/kasun/git-repos/testgrid"/>
        <property name="java.runtime.version" value="1.8.0_144-b01"/>
        <property name="java.awt.graphicsenv" value="sun.awt.X11GraphicsEnvironment"/>
        <property name="java.endorsed.dirs" value="/home/kasun/Applications/jdk1.8.0_144/jre/lib/endorsed"/>
        <property name="os.arch" value="amd64"/>
        <property name="java.io.tmpdir" value="/tmp"/>
        <property name="line.separator" value="&#10;"/>
        <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
        <property name="os.name" value="Linux"/>
        <property name="classworlds.conf" value="/usr/share/maven/bin/m2.conf"/>
        <property name="sun.jnu.encoding" value="UTF-8"/>
        <property name="java.library.path" value="/usr/java/packages/lib/amd64:/usr/lib64:/lib64:/lib:/usr/lib"/>
        <property name="java.specification.name" value="Java Platform API Specification"/>
        <property name="java.class.version" value="52.0"/>
        <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
        <property name="os.version" value="4.13.0-45-generic"/>
        <property name="user.home" value="/home/kasun"/>
        <property name="user.timezone" value="Asia/Colombo"/>
        <property name="java.awt.printerjob" value="sun.print.PSPrinterJob"/>
        <property name="file.encoding" value="UTF-8"/>
        <property name="java.specification.version" value="1.8"/>
        <property name="user.name" value="kasun"/>
        <property name="java.class.path" value="/usr/share/maven/boot/plexus-classworlds-2.x.jar"/>
        <property name="java.vm.specification.version" value="1.8"/>
        <property name="sun.arch.data.model" value="64"/>
        <property name="java.home" value="/home/kasun/Applications/jdk1.8.0_144/jre"/>
        <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher clean install"/>
        <property name="java.specification.vendor" value="Oracle Corporation"/>
        <property name="user.language" value="en"/>
        <property name="awt.toolkit" value="sun.awt.X11.XToolkit"/>
        <property name="java.vm.info" value="mixed mode"/>
        <property name="java.version" value="1.8.0_144"/>
        <property name="java.ext.dirs" value="/home/kasun/Applications/jdk1.8.0_144/jre/lib/ext:/usr/java/packages/lib/ext"/>
        <property name="securerandom.source" value="file:/dev/./urandom"/>
        <property name="sun.boot.class.path" value="/home/kasun/Applications/jdk1.8.0_144/jre/lib/resources.jar:/home/kasun/Applications/jdk1.8.0_144/jre/lib/rt.jar:/home/kasun/Applications/jdk1.8.0_144/jre/lib/sunrsasign.jar:/home/kasun/Applications/jdk1.8.0_144/jre/lib/jsse.jar:/home/kasun/Applications/jdk1.8.0_144/jre/lib/jce.jar:/home/kasun/Applications/jdk1.8.0_144/jre/lib/charsets.jar:/home/kasun/Applications/jdk1.8.0_144/jre/lib/jfr.jar:/home/kasun/Applications/jdk1.8.0_144/jre/classes"/>
        <property name="java.vendor" value="Oracle Corporation"/>
        <property name="maven.home" value="/usr/share/maven"/>
        <property name="file.separator" value="/"/>
        <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
        <property name="sun.cpu.endian" value="little"/>
        <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
        <property name="sun.desktop" value="gnome"/>
        <property name="sun.cpu.isalist" value=""/>
    </properties>
    <testcase name="testPrintSummary" classname="org.wso2.testgrid.core.TestPlanExecutorTest" time="0.09"/>
    <testcase name="testExecute" classname="org.wso2.testgrid.core.command.GenerateTestPlanCommandTest" time="0.901"/>
    <testcase name="testExecute" classname="org.wso2.testgrid.core.command.GenerateTestPlanCommandTest" time="0.026"/>
    <testcase name="testExecute" classname="org.wso2.testgrid.core.command.RunTestPlanCommandTest" time="1.465"/>
    <testcase name="testExecute" classname="org.wso2.testgrid.core.command.RunTestPlanCommandTest" time="0.105"/>
    <testcase name="testGetCombinations" classname="org.wso2.testgrid.infrastructure.InfrastructureCombinationsProviderTest" time="0.023"/>
    <testcase name="testValidateAddAPIsWithDifferentCase" classname="org.wso2.am.integration.tests.other.APIMANAGER3226APINameWithDifferentCaseTestCase" time="0">
        <skipped/>
        <system-out><![CDATA[INFO  [org.wso2.carbon.automation.engine.testlisteners.TestManagerListener] - =================== Running the test method org.wso2.am.integration.tests.other.APIMANAGER3226APINameWithDifferentCaseTestCase.testValidateAddAPIsWithDifferentCase ===================
]]></system-out>
    </testcase>
    <testcase name="destroy" classname="org.wso2.am.integration.tests.other.APIMANAGER3226APINameWithDifferentCaseTestCase" time="9.081">
        <failure type="org.wso2.am.integration.test.utils.APIManagerIntegrationTestException">org.wso2.am.integration.test.utils.APIManagerIntegrationTestException: Unable to get API - echo. Error: Read timed out
            at java.net.SocketInputStream.socketRead0(Native Method)
            at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
            at java.net.SocketInputStream.read(SocketInputStream.java:171)
            at java.net.SocketInputStream.read(SocketInputStream.java:141)
            at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:136)
            at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:152)
            at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:270)
            at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
            at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
            at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:260)
            at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:161)
            at sun.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
            at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
            at java.lang.reflect.Method.invoke(Method.java:498)
            at org.apache.http.impl.conn.CPoolProxy.invoke(CPoolProxy.java:138)
            at com.sun.proxy.$Proxy10.receiveResponseHeader(Unknown Source)
            at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:271)
            at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:123)
            at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:253)
            at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:194)
            at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:85)
            at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:186)
            at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
            at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:106)
            at org.wso2.am.integration.test.utils.http.HTTPSClientUtils.sendPOSTMessage(HTTPSClientUtils.java:210)
            at org.wso2.am.integration.test.utils.http.HTTPSClientUtils.doPost(HTTPSClientUtils.java:78)
            at org.wso2.am.integration.test.utils.http.HTTPSClientUtils.doPost(HTTPSClientUtils.java:150)
            at org.wso2.am.integration.test.utils.clients.APIPublisherRestClient.deleteAPI(APIPublisherRestClient.java:238)
            at org.wso2.am.integration.tests.other.APIMANAGER3226APINameWithDifferentCaseTestCase.destroy(APIMANAGER3226APINameWithDifferentCaseTestCase.java:98)
        </failure>
        <system-out><![CDATA[INFO  [org.wso2.carbon.automation.engine.testlisteners.TestManagerListener] - =================== On test success org.wso2.am.integration.tests.other.APIMANAGER3226APINameWithDifferentCaseTestCase.testValidateAddAPIsWithDifferentCase ===================
INFO  [org.wso2.carbon.automation.extensions.servers.utils.ServerLogReader] - [2018-07-17 19:43:51,139]  INFO - DependencyTracker Sequence : _auth_failure_handler_ was added to the Synapse configuration successfully
INFO  [org.wso2.carbon.automation.extensions.servers.utils.ServerLogReader] - [2018-07-17 19:43:51,139]  INFO - SequenceDeployer Sequence: _auth_failure_handler_ has been updated from the file: /opt/wso2/workspace/product-apim/modules/integration/tests-integration/tests-backend/target/carbontmp1531852456771/wso2am-2.5.0/repository/deployment/server/synapse-configs/default/sequences/_auth_failure_handler_.xml
INFO  [org.wso2.carbon.automation.extensions.servers.utils.ServerLogReader] - [2018-07-17 19:43:51,251]  INFO - CarbonAuthenticationUtil 'admin@carbon.super [-1234]' logged in at [2018-07-17 19:43:51,251+0000]
INFO  [org.wso2.carbon.automation.extensions.servers.utils.ServerLogReader] - [2018-07-17 19:43:53,142]  INFO - DependencyTracker Sequence : _build_ was added to the Synapse configuration successfully
INFO  [org.wso2.carbon.automation.extensions.servers.utils.ServerLogReader] - [2018-07-17 19:43:53,142]  INFO - SequenceDeployer Sequence: _build_ has been updated from the file: /opt/wso2/workspace/product-apim/modules/integration/tests-integration/tests-backend/target/carbontmp1531852456771/wso2am-2.5.0/repository/deployment/server/synapse-configs/default/sequences/_build_.xml
INFO  [org.wso2.carbon.automation.extensions.servers.utils.ServerLogReader] - [2018-07-17 19:43:55,143]  INFO - DependencyTracker Sequence : _production_key_error_ was added to the Synapse configuration successfully
INFO  [org.wso2.carbon.automation.extensions.servers.utils.ServerLogReader] - [2018-07-17 19:43:55,143]  INFO - SequenceDeployer Sequence: _production_key_error_ has been updated from the file: /opt/wso2/workspace/product-apim/modules/integration/tests-integration/tests-backend/target/carbontmp1531852456771/wso2am-2.5.0/repository/deployment/server/synapse-configs/default/sequences/_production_key_error_.xml
INFO  [org.wso2.carbon.automation.extensions.servers.utils.ServerLogReader] - [2018-07-17 19:43:57,144]  INFO - DependencyTracker Sequence : _resource_mismatch_handler_ was added to the Synapse configuration successfully
INFO  [org.wso2.carbon.automation.extensions.servers.utils.ServerLogReader] - [2018-07-17 19:43:57,144]  INFO - SequenceDeployer Sequence: _resource_mismatch_handler_ has been updated from the file: /opt/wso2/workspace/product-apim/modules/integration/tests-integration/tests-backend/target/carbontmp1531852456771/wso2am-2.5.0/repository/deployment/server/synapse-configs/default/sequences/_resource_mismatch_handler_.xml
]]></system-out>
    </testcase>
    <testcase name="destroy" classname="org.wso2.am.integration.tests.other.APIMANAGER3226APINameWithDifferentCaseTestCase" time="0.001">
        <failure message="Access token response contains errors. expected:&lt;false&gt; but was:&lt;true&gt;" type="java.lang.AssertionError">java.lang.AssertionError: Access token response contains errors. expected:&lt;false&gt; but was:&lt;true&gt;
            at org.testng.Assert.fail(Assert.java:89)
            at org.testng.Assert.failNotEquals(Assert.java:489)
            at org.testng.Assert.assertFalse(Assert.java:58)
            at org.wso2.identity.integration.test.oauth2.OAuth2IDTokenEncryptionTestCase.testAuthCodeGrantSendGetTokensPost(OAuth2IDTokenEncryptionTestCase.java:253)
        </failure>
        <system-out><![CDATA[INFO  [org.wso2.carbon.automation.engine.testlisteners.TestManagerListener] - =================== On test skipped org.wso2.am.integration.tests.other.APIMANAGER3226APINameWithDifferentCaseTestCase.testValidateAddAPIsWithDifferentCase ===================
]]></system-out>
    </testcase>
</testsuite>